bundle/config/mutator/python/python_locations.go

```diff
@@ -0,0 +1,163 @@
+package python
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"path/filepath"
+
+	"github.com/databricks/cli/libs/dyn"
+)
+
+const generatedFileName = "__generated_by_pydabs__.yml"
```

Comment:
Please add a comment explaining why we need this const, or where it is used.

Reply:
👍 , added above